config.toml

```diff
@@ -156,12 +156,19 @@ js = [
   "script"
 ]
 
+[[params.versions]]
+fullversion = "v1.19.0"
+version = "v1.19"
+githubbranch = "v1.19.0"
+docsbranch = "release-1.19"
+url = "https://kubernetes.io"
+
 [[params.versions]]
 fullversion = "v1.18.0"
 version = "v1.18"
 githubbranch = "v1.18.0"
 docsbranch = "release-1.18"
-url = "https://kubernetes.io"
+url = "https://v-18.docs.kubernetes.io"
```

Comment:
@savitharaghunathan , would you check this url?

Reply:
Hi @kbhawkey, Thanks for catching it. I created PR to fix it - https://github.com/kubernetes/website/pull/23460 Can you take a look at it? I also fixed k8s minor versions along with it

Reply:
Will wait for the fix to merge and then resolve this